src/app/_components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { ChevronRightIcon } from "@heroicons/react/20/solid";
+
+interface BreadCrumbProps {
+  items: Array<{
+    name: string;
+    href?: string;
+    current?: boolean;
+    onClick?: () => void;
```

Comment:
Don't think onClick should be apart of the items array. Probably should be it's own prop.